cmd/root.go

```diff
@@ -194,6 +196,21 @@ func extractEnvVars(inputEnvVars []string) []ECS.EnvVar {
 	return envVars
 }
 
+func convertEnvVarKeyValuesToECSEnvVars(inputEnvVars []string, envVarFile string) []ECS.EnvVar {
```

Comment:
Where is this function used? I don't see a call anywhere in the source tree..

Reply:
Nowhere. Good catch. Just removed it.